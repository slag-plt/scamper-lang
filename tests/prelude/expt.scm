(expt 7 2)
(expt 0.2 0.5)
(expt 100 1)
(expt 2.0 2)
(expt 3.0 3.1)