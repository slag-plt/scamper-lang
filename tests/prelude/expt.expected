49
0.4472135954999579
100
4
30.135325698915423
